screens/Onboarding/styles.ts

```diff
@@ -1,88 +1,71 @@
-import { StyleSheet } from 'react-native' 
+import { StyleSheet } from "react-native"
 import {
   white,
   black,
   lightGray,
   lightPeach,
   peach,
   shadowColor,
-} from '../../assets/colors/colors' 
+} from "../../assets/colors/colors"
 
 import {
   BUTTON_HEIGHT,
   BUTTON_RADIUS,
   BUTTON_TEXT_FONT_SIZE,
-} from '../../assets/global/constants' 
+} from "../../assets/global/constants"
 
 const styles = StyleSheet.create({
-  border: {
-    flex: 1,
-    flexDirection: 'column',
-    justifyContent: 'space-between',
-  },
   button: {
-    alignItems: 'center',
+    alignItems: "center",
     backgroundColor: lightPeach,
     borderColor: peach,
     borderRadius: BUTTON_RADIUS,
     borderWidth: 2,
     elevation: 8,
     height: BUTTON_HEIGHT,
-    justifyContent: 'center',
+    justifyContent: "center",
     marginBottom: 10,
     shadowColor,
     shadowOffset: { width: 0, height: 4 },
     shadowOpacity: 0.3,
     shadowRadius: 5,
-    width: '100%',
+    width: "100%",
   },
   buttonGoogle: {
-    alignItems: 'center',
     backgroundColor: white,
     borderRadius: BUTTON_RADIUS,
     borderWidth: 1,
-    display: 'flex',
-    flexDirection: 'row',
+    display: "flex",
```

Comment:
i am pretty sure that if you delete `display: "flex"`, the button is going to look the same way

Reply:
Indeed